src/scss/components/_serp-header.scss

```diff
@@ -0,0 +1,32 @@
+/**
+ * SERP header Component
+ * =================================
+ * Used for restaurant count and location on search page
+ *
+ */
+
+.c-serp-header {
```

Comment:
Would be good if we could name this to be more generic and not related to SERP.  It may be resuable on another page where there's a similar pattern and the `c-serp-header` would be a little strange.

Reply:
If this is just a generic header, could just call this something like `c-contentHeader`